src/GeositeFramework/js/Plugin.js

```diff
@@ -638,13 +638,21 @@ require(['use!Geosite',
                     mapReadyDeferred.resolve(map);
 
                     $('#print-modal-confirm').on('click', function() {
+                        // Before moving the map, get the current center and extent
+                        var center = map.extent.getCenter();
+                        var extent = map.extent;
```

Comment:
Could probably drop the comment here if these were named something like `previousMapCenter` and `previousMapExtent`. Not sure whether that'd be more readable though.